sound/soc/bcm/allo-katana-codec.c

```diff
@@ -236,7 +242,22 @@ static int katana_codec_set_fmt(struct snd_soc_dai *dai, unsigned int fmt)
 	return 0;
 }
 
+int katana_codec_dai_mute_stream(struct snd_soc_dai *dai, int mute, int stream)
+{
+	struct snd_soc_codec *codec = dai->codec;
+	struct katana_codec_priv *katana_codec = snd_soc_codec_get_drvdata(codec);
```

Comment:
Even though this is downstream code we try to stick to kernel coding conventions - please split this long line,